homeassistant/components/airvisual_pro/sensor.py

```diff
@@ -45,66 +45,74 @@ class AirVisualProMeasurementDescription(
         key="air_quality_index",
         device_class=SensorDeviceClass.AQI,
         state_class=SensorStateClass.MEASUREMENT,
-        value_fn=lambda settings, status, measurements: measurements[
+        value_fn=lambda settings, status, measurements, history: measurements[
             async_get_aqi_locale(settings)
         ],
     ),
+    AirVisualProMeasurementDescription(
+        key="outdoor_air_quality_index",
+        device_class=SensorDeviceClass.AQI,
+        state_class=SensorStateClass.MEASUREMENT,
+        value_fn=lambda settings, status, measurements, history: history.get(
+            f'Outdoor {"AQI(US)" if settings["is_aqi_usa"] else "AQI(CN)"}', None
```

Comment:
`.get` will already default to None, so you don't need to explicitly add it here.